ddp-workspace/projects/ddp-pancan/src/app/components/faq-section/faq-section.component.html

```diff
@@ -1,6 +1,6 @@
-<mat-accordion hideToggle="true" multi="true" displayMode="flat">
+<mat-accordion hideToggle="true" multi="true" displayMode="flat" >
     <ng-container *ngFor="let item of questions | translate | slice : 0 : questionsCount;">
-        <mat-expansion-panel #panel>
+        <mat-expansion-panel #panel [class.darkTheme]="darkTheme">
```

Comment:
Could you please clarify if we can't just simply use dark grey divider lines for both types of background? we can get rid of darkTheme switcher then

Reply:
On mockups borders are different for pages.
Can't clarify it for now (Erin is on vacation).

Reply:
let's add todo then

Reply:
added.